### PR TITLE
fix(material-experimental/mdc-paginator): target page size label with `aria-labelledby`

### PR DESCRIPTION
**Pull request by zarend**
See commit message for description.

Original bug is filed internally at http://b/188218083

From the bug report:

ENVIRONMENT
OS | Browser | Assistive Technology    
Windows version 20H2,  Firefox version 88.0.1 (64-bit)
JAWS 2021 Version 2021.2102.34 ILM,
NVDA version 2020.4

PREREQUISITES
 Turn on screen reader
 
STEPS TO REPRODUCE
Open Mozilla Firefox
Navigate to https://xap-gallery.googleplex.com/embed/a11y/mdc-paginator
Tab to the number of items combo box.  
Notice the announcement.

EXPECTED RESULT(S)
the value of the combo box should be announced by the screen reader.

ACTUAL RESULT(S)
The screen reader announces the role of the combo box, but not the value .  e.g. “Items per page: Combo box, To change the selection use the Arrow keys”.
 
ADDITIONAL NOTE(S)
This issue is only reproducible in Mozilla Firefox.
 
VIDEO/SCREENSHOT(S)
https://drive.google.com/file/d/1HRDaA8bXKXlduobTUmWUraAUd7v1tZ5h/view?usp=sharing&resourcekey=0-rNDyr76TRjgPc9mEQH-exg

### CODE REVIEW

**Comment by zarend** [comment]
Kicked off a [presubmit](http://cl/384804567)

**Comment by jelbourn** [comment]
@zarend were you able to test this fix on Windows and confirm it addresses the issues?

**Comment by zarend** [comment]
Yes, I validated that this fixes the bug on Windows Firefox using the mdc-paginator demo in the dev-app.

File: src/material-experimental/mdc-paginator/paginator.ts [hunk 1]
```diff
@@ -66,6 +68,10 @@ export class MatPaginator extends _MatPaginatorBase<MatPaginatorDefaultOptions>
   /** If set, styles the "page size" form field with the designated style. */
   _formFieldAppearance?: MatFormFieldAppearance;
 
+  /** ID for the DOM node containing the pagiators's items per page label. */
+  readonly _pageSizeLabelId = `mat-paginator-page-size-label-${nextUniqueId++}`;
```
Comment (jelbourn) [inline src/material-experimental/mdc-paginator/paginator.ts:72]: I might shorten this to something like `mat-pgntr-x` just to shave off a few bytes

**Review by jelbourn — APPROVED** [review]
LGTM
